### PR TITLE
Remove repositories

### PR DESCRIPTION
**Pull request by Riari**
As per #33.

### CODE REVIEW

**Comment by NowNewStart** [comment]
Beautiful

File: src/Controllers/DiscussionController.php [hunk 1]
```diff
@@ -49,9 +32,10 @@ public function store(CreateDiscussionRequest $request)
      */
     public function update(UpdateDiscussionRequest $request, $id)
     {
-        $update = $this->discussionRepository->updateDiscussion($request->all());
+        $discussion = Discussion::find($id);
```
Comment (deleugpn) [inline src/Controllers/DiscussionController.php]: What about leveraging the Implicit Route Binding here?

Comment (Riari) [inline src/Controllers/DiscussionController.php]: Separate concern IMO and I already opened an issue about it: #40

Comment (Dylan-DPC) [inline src/Controllers/DiscussionController.php]: Better to change it once for all right?

Comment (AndreasElia) [inline src/Controllers/DiscussionController.php]: Seems unrelated to this PR, should probably do it seperately.

Comment (Riari) [inline src/Controllers/DiscussionController.php]: I just replaced the repository lines with non-repo equivalents. For anything other than creating, currently, that means two lines instead of one (one to find, then another to do the operation). I know it's possible to do something like `->whereId($id)->update($request->all())`, but the return value of that isn't a model AFAIK, so it can't be used for the response.

No doubt this stuff will change in a later PR anyway since we're thinking about using route model binding.

**Comment by AndreasElia** [comment]
Please pull latest StyleCI fixes from master.
